.ci/build

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env sh
+
+set -e
+set -x
+
+# $1 SRC_DIR (repository root directory)
+# $2 VERSION_FILE (a regular file containing the build version)
+# OR via env vars:
+# SOURCE_PATH
+# VERSION_PATH
+
+if [ ! -z "$SOURCE_PATH" ]; then
+  export SRC_DIR="${SOURCE_PATH}"
+else
+  own_dir="$(dirname $(readlink -f ${0}))"
+  export SRC_DIR="${own_dir}/.."
+fi
+export SRC_DIR="$(readlink -f "${SRC_DIR}")"
+if [ ! -z "$VERSION_PATH" ]; then
+  export VERSION_FILE="${VERSION_PATH}/number"
+else
+  export VERSION_FILE="$(readlink -f "${SRC_DIR}/VERSION")"
+fi
+
+echo "SRC_DIR: ${SRC_DIR}"
+
+export VERSION="$(cat "${VERSION_FILE}")"
+
+export OUT_DIR="$(readlink -f "${BINARY_PATH}")/bin"
+echo "OUT_DIR: ${OUT_DIR}"
+
+# dance around go-build
+export GOPATH="${SRC_DIR}"
+mkdir -p "${SRC_DIR}/src/github.com/gardener"
+src_symlink="${SRC_DIR}/src/github.com/gardener/node-controller-manager"
```

Comment:
we have changed the name to  "machine-controller-manager", can you please update accordingly.